homeassistant/components/onewire/sensor.py

```diff
@@ -54,7 +58,20 @@
 class OneWireSensorEntityDescription(OneWireEntityDescription, SensorEntityDescription):
     """Class describing OneWire sensor entities."""
 
-    override_key: str | None = None
+    override_key: Callable[[str, Mapping[str, Any]], str] | None = None
+
+
+def _get_sensor_precision(device_id: str, options: Mapping[str, Any]) -> str:
+    """Get precision form config flow options."""
```

Comment:
I think this should be more specific. It should be specific for the model (ds18b20) or the family (28).

Reply:
Done